docs/Catalog/Packaging.md

```diff
@@ -0,0 +1,226 @@
+# Packaging Engine
+
+## Overview
+
+The Quilt Packaging Engine in the Quilt Platform allows administrators and
+developers to automate the process of creating Quilt packages from data stored
+in Amazon S3. It serves as a key component in Quilt's SDMS (Scientific Data
+Management System) strategy, enabling automated data ingestion and
+standardization. It consists of:
```

Comment:
Suggest "currently consists of" implying that there are features to come down the line.